rebar.config

```diff
@@ -1,14 +1,57 @@
+%% -----------------------------------------------------------------------------
+%%
+%% rebar.config: SQL - rebar3 configuration file.
+%%
+%% Copyright (c) 2012-17 K2 Informatics GmbH.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -----------------------------------------------------------------------------
+
 {cover_enabled, true}.
-{cover_excl_mods, [sql_lex, sqlparse]}.
+{cover_excl_mods, [
+    sql_lex,
+    sqlparse,
+    sqlparse_generator
+]}.
+{cover_opts, [verbose]}.
+
+{ct_opts, [
+    {cover},
+    {verbose}
+]}.
 
-{deps, [{jpparse, {git, "https://github.com/K2InformaticsGmbH/jpparse.git", {tag, "1.1.0"}}}]}.
+{deps, [{jpparse, {git, "https://github.com/K2InformaticsGmbH/jpparse.git", {tag, "1.2.0"}}}]}.

```

Comment:
Please change to `{branch,"master"}` or travis to pass.
We change to `{tag, _version_}` after tagging before release

Reply:
Fixed.